src/components/ExtraInfoModal.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <button v-bind="attrs" type="button" @click.stop.prevent="show = true">
+    <p-icon icon="QuestionMarkCircleIcon" solid />
```

Comment:
A little padding would let the text and ? icon breath, maybe just a `pl-1` or `pl-2` class